README.md

```diff
@@ -5,6 +5,8 @@ chat](https://badges.gitter.im/open-telemetry/opentelemetry-dotnet.svg)](https:/
 [![Build
 Status](https://action-badges.now.sh/open-telemetry/opentelemetry-dotnet)](https://github.com/open-telemetry/opentelemetry-dotnet/actions)
 [![Release](https://img.shields.io/github/v/release/open-telemetry/opentelemetry-dotnet?include_prereleases&style=)](https://github.com/open-telemetry/opentelemetry-dotnet/releases/)
+[![Nuget](https://img.shields.io/nuget/vpre/OpenTelemetry.svg)](https://www.nuget.org/profiles/OpenTelemetry)

```

Comment:
SHould we do this in separate lines? with some hint to users that myget is daily build, nuget if official release, or something?

Reply:
We can do something like:
- stable/nuget release: nugetbadge
- beta/myget release: mygetbadge

i wouldn't add the daily, since its a merge. What do you  think?

Reply:
this can also cause confusion as to the nuget one is a stable one. It can be confused with GA.
will come back.

Reply:
and, with that, don't know if we need the release badge.